src/blocks/Listing/components/ListingTwoColumns.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react'
```

Comment:
Two columns is not a type. It's just and story / example on the code. Please remove the variant and keep the example on the stories

Reply:
Done, two column merged to base